.github/workflows/wee-slack.yml

```diff
@@ -0,0 +1,103 @@
+name: wee-slack
+on: [push, pull_request]
+
+jobs:
+  lint:
+    if: >
+      github.event_name == 'push' || (
+        github.event_name == 'pull_request' &&
+        github.event.pull_request.head.repo.full_name != github.repository
```

Comment:
What is this condition for?

Reply:
I'm assuming this is for when the PR is from a fork, not this repository.

Reply:
Yeah, GitHub Actions has an issue in which specifying [push, pull_request] events will:run the workflow for:

* Any push to any branch on the main repository
* Any pull request or subsequent update to a pull request

This unfortunately causes any PR created from a branch in the main repository to run the jobs for both events. This check bypasses that.

Reply:
Ah, okay.